### PR TITLE
test: add selenium acceptance tests to nightly build - Bonitoo/selenium nightly

### PR DESCRIPTION
**Pull request by karel-rehor**
Closes #

Adds a selenium_accept job to circleci and adds it to the nightly workflow.  

- [ ] [CHANGELOG.md](https://github.com/influxdata/influxdb/blob/master/CHANGELOG.md) updated with a link to the PR (not the Issue)
- [x] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [ ] Rebased/mergeable
- [ ] Tests pass
- [ ] http/swagger.yml updated (if modified Go structs or API)
- [ ] Documentation updated or issue created (provide link to issue/pr)
- [x] Signed [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -80,6 +80,45 @@ jobs:
...
+           cp ~/project/selenium-accept-infl2/report/cucumber_report.html ~/artefacts/html/cucumber_report.html
+           cp ~/project/selenium-accept-infl2/report/cucumber_junit.xml ~/test-results/cucumber/report.xml
+           cp ~/project/selenium-accept-infl2/report/cucumber_junit.xml ~/artefacts/report.xml
+           cp -r ~/project/selenium-accept-infl2/screenshots ~/artefacts
+           ls -al
+           exit $TEST_RESULT
+      - store_test_results:
+          path: ~/test-results
+      - store_artifacts:
+          path: ~/artefacts
```
Comment (drdelambre) [inline .circleci/config.yml]: should be `artifacts`

Comment (karel-rehor) [reply .circleci/config.yml]: renamed to artifacts

File: .circleci/config.yml [hunk 2]
```diff
@@ -80,6 +80,45 @@ jobs:
...
+         name: Checkout Tests
+         command: git clone --single-branch --branch 2.0-Alpha https://github.com/bonitoo-io/selenium-accept-infl2.git
+      - run:
+         name: Selenium tests
+         command: |
+           set +e
+           cd selenium-accept-infl2
+           npm install
+           npm test; TEST_RESULT=$?
+           sleep 30
```
Comment (gshif) [inline .circleci/config.yml]: What is the purpose of `sleep`?

Comment (karel-rehor) [reply .circleci/config.yml]: Earlier in the week some data-testid's were changed, which lead to waits on elements to hang beyond the end of the test, which then lead to cucumber not writing out reports.  The sleep was added while troubleshooting the issue. I kept it as a kind of insurance, but have now removed it.  

I just added a line using curl to ensure the endpoint is up and can be used before proceeding with tests.

**Pull request merged**